### PR TITLE
Add more go doc

### PR DESCRIPTION
**Pull request by AntoineThebaud**

### CODE REVIEW

**Review by Nexucis — CHANGES_REQUESTED** [review]
I would prefer that the package description is actually a comments for the corresponding package.

Like that we can also have the golang documention updated in pkg.go.dev

**Comment by AntoineThebaud** [comment]
Changed README files by additional go doc where relevant

File: etcd/client.go [hunk 1]
```diff
@@ -11,6 +11,8 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+// This package provides a simple DAO to ease the interaction with ETCD. It also provides a lock system.
```
Comment (Nexucis) [inline etcd/client.go:14]: In the go doc, you should start by `Package etcd ....`.

Comment (Nexucis) [reply etcd/client.go:14]: And I think you can remove the extra line between the comment and the name of the package :)

File: slices/contains.go [hunk 2]
```diff
@@ -11,6 +11,8 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+// This package provides utility methods to manipulate slices (mostly slices of string).
```
Comment (Nexucis) [inline slices/contains.go:14]: In the go doc, you should start by `Package slices.....`

**Review by Nexucis — CHANGES_REQUESTED** [review]
Excepting two tiny comments, it's really cool !!
Thanks for doing it :)

**Comment by Nexucis** [comment]
nice ! thanks @AntoineThebaud :)

**Pull request merged**